apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx

```diff
@@ -194,18 +194,28 @@ const ValidatedInstructions: React.FunctionComponent<InstructionsProps> = ({
 
   // There are 3 ways to "meet validation" for a level:
   // If the level is a predict level, the user must run the code.
-  // Otherwise, if the level has conditions, they must be satisfied.
-  // If the level has no conditions and is not a predict level,
-  // the user must run their code at least once.
+  // If the level has conditions, they must be satisfied.
```

Comment:
Super nit: are there now 4 ways to 'meet validation'?

Reply:
updated!